docs/user/exercises/textual.rst

```diff
@@ -189,6 +189,92 @@ Once you're done assessing the solution, you can either:
 
 - Click on |exercise-dashboard-button| to navigate to the **Exercise Dashboard**.
 
+Automatic Student Feedback
+--------------------------
+
+.. admonition:: Why Automatic Student Feedback
+   :class: tip
+
+   In large courses, providing timely and personalized feedback on text-based exercises is 
+   challenging. Automated student feedback helps learners identify misunderstandings early, 
+   iterate on their drafts, and refine their writing skills—all without waiting for an 
+   instructor or tutor to be available.
+
+**Overview:**
+
+When a text exercise is configured with the ``Allow automatic AI preliminary feedback requests`` 
+option, students can request preliminary AI feedback on their written submissions. 
+The feedback is generated through the :ref:`Athena Service <athena_service>`, 
+which analyzes the **content**, **structure**, and **style** of the text and produces 
+comments based on the provided **Grading Instructions**, **Problem Statement**, and **Sample Solution**.
+
+.. admonition:: Note
+   :class: note
+
+   For best results, include detailed **Grading Instructions** as **Structured Grading Instructions**, and provide a Sample Solution (optional but recommended). This ensures that AI-generated feedback aligns with the intended evaluation criteria and offers focused, actionable suggestions.
+
+**How to Request Automatic Feedback:**
+
+1. **Requesting Feedback**
+
+   .. container::
+
+      - Navigate to a **Text Exercise** with the **Automatic Student Feedback** feature enabled.
+      - Enter your response in the text editor and submit it.
+      - You can request AI feedback either from the exercise overview page or directly within the text editor.
+
```

Comment:
Here might make sense to mention for each exercise students have 10 feedback limit